packages/components/src/LabelledMessage/LabelledMessage.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import styles from "./LabelledMessage.module.scss"
+
+export interface LabelledMessageProps {
+  label: string | React.ReactElement
+  message: string | React.ReactElement
+}
+
+export const LabelledMessage = ({
```

Comment:
Why is this considered a standalone component? Maybe belongs in `Filter`?

Reply:
I guess the use case is too generic that I didn't feel it made sense to belong in there 😬 We could use it for any text with the format of `{label}:{message}` (maybe even allow changing the separator later). It's moreso a i18n helper 😬 

We could actually use it for FilterButton too that I look at it lol. But any place that has that pattern it can be used. Might be useful for consumers too?

Reply:
Perhaps it needs to be named `Format*` component or maybe `use*` to make it a hook... Just feels like a lost component without something to make it sound more like a utility or helper, cause you can't really just use it by itself outside the context of another component.

Reply:
Sure you can :D Who says you can't just stick it on your page as contents? Lol

Reply:
Lol okay. But what if I wanted to use this to label a list of `Tag` components? `Message` would be incorrect 😄

Reply:
Hmm... What do you propose that part be called then? It's a labelled something. I don't want to name it `Item` because I have that associated with Select 😂 LabelledNode? LabelledText?

Reply:
`LabelledList`? A list can be of a singular item 😈

Reply:
That is 100% worse than `Message` 😂

Reply:
Damn, I was aiming for 150% worse! Let's go with `Message` 😄